src/hotspot/share/utilities/filterQueue.hpp

```diff
@@ -67,10 +67,10 @@ class FilterQueue {
   template <typename MATCH_FUNC>
   bool contains(MATCH_FUNC& match_func);
```

Comment:
Need to update the copyright for this file.

Reply:
Fixed